proto/process/agent.proto

```diff
@@ -465,3 +465,35 @@ message Host {
 message DNSEntry {
   repeated string names = 1;
 }
+
+// reference https://github.com/kubernetes/kubernetes/blob/cb19b56831d54d1d31249949318ef0b07bf00df9/staging/src/k8s.io/api/core/v1/generated.proto
+message Pod {
+	string name = 1;
+	string namespace = 2;
+	int64 age = 3;
+	string IP = 4;
+	string nominatedNodeName = 5;
```

Comment:
would it be better to use the node UUID? My understanding is that it's technically possible for node names to be re-used within  an org if you have multiple kubernetes clusters, and that would make getting the node for a pod tricky.

Reply:
Good point, I don't think we have this info at the pod level though, with `cluster-name` + `hostname` we should be fine though 🤔 (although I'm not sure how we can enforce `cluster-name` to be unique between clusters)